assets/app/views/_templateopt.html

```diff
@@ -29,6 +30,21 @@
           autocapitalize="off"
           spellcheck="false"
           ng-attr-aria-describedby="{{parameter.description ? (paramID + '-description') : undefined}}">
+      <textarea
+          ng-if="parameter.multiline"
+          ng-attr-id="{{paramID}}"
+          ng-attr-name="{{paramID}}"
+          class="form-control"
+          rows="4"
+          placeholder="{{ parameter | parameterPlaceholder }}"
+          ng-model="parameter.value"
+          ng-required="parameter.required && !parameter.generate"
+          ng-blur="cleared = false"
+          ng-trim="false"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
```

Comment:
Yeah I'll remove them, it could theoretically be used for someone to put in a description or something